strings/names/ninjaname.txt

```diff
@@ -5,7 +5,6 @@ Bro
 Cyrax
 Daemon
 Death
-Donatello
```

Comment:
most of the ninja names are references in some way, i dont think deleting just the ones you get is necessary

Reply:
Also ninja names are compound and theres no risk of an admin bwoinking someone for them anyway. (Still should remove the OOC one (McAwesome))